perfzero/lib/benchmark.py

```diff
@@ -69,12 +71,43 @@ def _setup(self):
     self.benchmark_execution_time['checkout_repository'] = (
         time.time() - start_time)
 
+    # Start cloud TPU.
+    if self.config.tpu_parameters is not None:
+      start_time = time.time()
```

Comment:
Better to move L77 to L94 into 

utils.setup_tpu(self.config.tpu_parameters)

and finally clear it with 
utils.cleanup_tpu(self.config.tpu_parameters)

Reply:
Sure, moved to setup_tpu and cleanup_tpu functions in utils.